index.Rmd

```diff
@@ -237,9 +235,9 @@ Now this object is stored on your computer, ready to be archived for later use.
 Performing simple raster operations with a SpatRaster is fairly easy. For instance, if you want to subtract two SpatRasters of same extent, `r1` and `r2`; simply doing `r1 - r2` will give the expected output, which is, every pixel value of `r2` will be subtracted from the matching pixel value of `r1`. These types of pixel-based operations almost always require a set of conditions to be met in order to be executed; the two SpatRasters need to be identical in term of extent, pixel size, coordinate reference system, etc.
 
 ## Subsetting layers from SpatRaster
-Different spectral bands of the same satellite scene are often stored in multi-layer objects. This means that you will very likely import them in your R working environment as one multi-layer SpatRaster. As a consequence, to perform calculations between these bands, you will have to write an expression referring to individual layers of the object. Referring to individual layers in a SpatRaster can be done by using double square brackets `[[]]`.
+Previously, we used data with already derived NDVI. Now, we will make use of the Landsat 7 spectral bands to create the NDVI SpatRaster ourselves. The different spectral bands of the same satellite scene are often stored in multi-layer objects. This means that you will very likely import them in your R working environment as one multi-layer SpatRaster. As a consequence, to perform calculations between these bands, you will have to write an expression referring to individual layers of the object. Referring to individual layers in a SpatRaster can be done by using double square brackets `[[]]`.
 
-Let's look for instance at how the famous NDVI index would have to be calculated from the *gewata* SpatRaster read earlier, and that contains the spectral bands of the Landsat 7 sensor. And in case you have forgotten, the NDVI formula is as follows.
+Let's look for instance at how the famous NDVI index would have to be calculated from the *gewata* SpatRaster read earlier. The *gewata* SpatRaster already contains [6 bands of the Landsat 7 sensor](https://www.usgs.gov/faqs/what-are-band-designations-landsat-satellites). The formula below uses Landsat 7's NIR and Red bands, corresponding to layers 4 and 3 of the gewata SpatRaster, respectively.
```

Comment:
Some more help for students who do not know much about remote sensing. Hopefully this section is not too much information at once.